tRecorderApi/api/views/helpers.py

```diff
@@ -34,3 +34,22 @@ def highPassFilter(location):
 def getRelativePath(location):
```

Comment:
method naming convention

Reply:
I think helpers.py is not being used anymore...